README.md

```diff
@@ -157,19 +157,33 @@ Build the docker image required using the dockerfile from this repository
 ``` bash
 git clone git@github.com:ros-swg/turtlebot3_demo.git
 cd turtlebot3_demo
-docker build . -t rosswg:turtlebot3_sandbox_demo
+docker build . -t ros-swg/turtlebot3_demo
```

Comment:
The docker repo for this github repo is `rosswg/turtlebot3_demo` , so we should probably be consistent on the latest tag we use to reference this Dockerfile, weather they build it themselves, or pull it from our Docker repo to skip the build time.

Reply:
Good catch! I was trying to change the existing readme to match the upstream tag - I messed it up, will change